.Gruntfile.babel.js

```diff
@@ -13,6 +13,7 @@ const url = require('url')
 const S = require('string')
 const vueify = require('vueify')
 const pathmodify = require('pathmodify')
+const envify = require('envify')
```

Comment:
I'm pretty sure this isn't necessary since we already have [`transform-inline-environment-variables`](https://github.com/okTurtles/group-income-simple/blob/master/.babelrc) which supposedly does the same thing.